lib/sql-string.js

```diff
@@ -66,6 +66,10 @@ function escape(val, timeZone, dialect, format) {
     throw new Error(`Invalid value ${util.inspect(val)}`);
   }
 
+  if (dialect === 'mysql' && val === 'CURRENT_TIMESTAMP') {
```

Comment:
Hardcoding specific values seems kind of hacky, we should probably handle this at a higher level or at least have a specific list.
I'm not an expect in this regard but aren't there other 'magic' default values too?

Reply:
See https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html